dispatcher/DAQController.py

```diff
@@ -361,6 +361,8 @@ def CheckTimeouts(self, detector, command = None):
                                          "can't solve"),
                                         'ERROR',
                                         "STOP_TIMEOUT")
+                    # also invoke the nuclear option
+                    self.hypervisor.TacticalNuclearOption()
```

Comment:
I've renamed this to `tactical_nuclear_option` in daqnt, sometimes I cannot help myself from having arbitrary preferences

Reply:
We can snake_case, sure